### ISSUE TITLE
v1.32.0 Release Notes: "Known Issues"

### ISSUE THREAD

**Issue by satyampsoni** [kubernetes/kubernetes#129009]
### What would you like to be added?

This issue is a bucket placeholder for collaborating on the "Known Issues" additions for the 1.32 Release Notes. If you know of issues or API changes that are going out in 1.32, please comment here so that we can coordinate incorporating information about these changes in the Release Notes.

/assign @kubernetes/release-team

/sig release
/milestone v1.32

### Why is this needed?

NA

**Comment by sftim** [comment]
I guess we can close this now?

**Comment by fsmunoz** [comment]
Yes, agreed, well past its time. This was addressed in https://github.com/kubernetes/sig-release/pull/2691 and https://github.com/kubernetes/sig-release/pull/2694 .

/close
